pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -1075,3 +1101,98 @@ func (c *K8sOrchestrator) GetNodesForVolumes(ctx context.Context, volumeIDs []st
 	}
 	return volumeIDToNodeNames
 }
+
+// initNodeIDToNameMap performs all the operations required to initialize
+// the node ID to  name map. It also watches for node add, update & delete
+// operations, and updates the map accordingly.
+func initNodeIDToNameMap(ctx context.Context) {
+	log := logger.GetLogger(ctx)
+	// If use-node-id is set then do not initialize the informer
+	if !k8sOrchestratorInstance.IsFSSEnabled(ctx, common.UseCSINodeId) {
```

Comment:
UseCSINodeId is already enabled in our CSI release and its part of the CSI 2.5. Given this, you don't need to handle this at all. 
UseCSINodeId will always be true.

Reply:
I added this check because the block vanilla tests were failing. I can check the clusterFlavor instead of this FSS check to ensure this is done only for wcp clusters.